### PR TITLE
Save tags in database as list

### PR DESCRIPTION
**Pull request by nddq**

### CODE REVIEW

File: Controllers/UploadController.py [hunk 1]
```diff
@@ -10,14 +10,14 @@
 def uploadNewFile():
     try:
         if ("SID" not in request.cookies):
-            return Response("No session detected", status=400)
+            return Response("No session detected", status=401)
         if ('file' not in request.files):
-            return Response("No files were uploaded", status=400)
+            return Response("No files were uploaded", status=402)
```
Comment (mdesilva) [inline Controllers/UploadController.py]: This should probably stay as a 400

Comment (nddq) [reply Controllers/UploadController.py]: Oops I forgot to change them back, were using them for debugging

File: Controllers/UploadController.py [hunk 2]
```diff
@@ -10,14 +10,14 @@
...
-            return Response("Prohibited file type", status=400) #TODO: Append to response: Dynamically return the types of allowed files
+            return Response("Prohibited file type", status=403) #TODO: Append to response: Dynamically return the types of allowed files
         
         dataset = UploadService.createDataSetAndDataObjects(request)
         
         return Response(str(dataset.id))
     except ValueError:
-        return Response("Empty fields detected. Please remove empty values from your dataset and try again.", status=400)
\ No newline at end of file
+        return Response("Empty fields detected. Please remove empty values from your dataset and try again.", status=404)
```
Comment (mdesilva) [inline Controllers/UploadController.py]: This should also be a 400

File: Services/UploadService.py [hunk 3]
```diff
@@ -7,6 +7,7 @@
 import pandas as pd
 import numpy
 import boto3
+import json
```
Comment (mdesilva) [inline Services/UploadService.py]: Do we need this import ?

**Pull request merged**